docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -832,6 +832,13 @@ You can see which parts of `s` were stored using the sparse serialization:
 └────────┴────────────────────┘
 ```
 
+## replace_long_file_name_to_hash {#ratio_of_defaults_for_sparse_serialization}
+If the file name for column is too long (more than `max_file_name_length` bytes) replace it to SipHash128. Default value: `false`.
```

Comment:
What are the implications on compatibility?

Reply:
As usual, it will be impossible to downgrade after enabling this